ngraph_bridge/ngraph_catalog.cc

```diff
@@ -43,10 +45,16 @@ string NGraphCatalog::CreateNodeKey(const int& graph_id,
   return to_string(graph_id) + "_" + node_name + ":" + to_string(index);
 }
 
+string NGraphCatalog::CreateNodeKey(const int& graph_id,
```

Comment:
best to rename it to avoid using this by mistake when the other was intended

Reply:
Both are doing the same thing but have different number of parameters. I would think that if someone intends to use the other one would provide the 3rd parameter, so overloading made more sense to me.

Reply:
I was talking about when someone intends to use the other one and misses sending the 3rd parameter, it will still work without the person realizing it is mistake.